graspologic/cluster/divisive_cluster.py

```diff
@@ -238,26 +268,86 @@ def _fit(self, X):
                             (labels, np.zeros((len(X), 1), dtype=int))
                         )
                     labels[inds, 1 : child_labels.shape[1] + 1] = child_labels
+                else:
+                    # make a "GaussianMixture" model for clusters
+                    # that were not fitted
+                    if self.cluster_method == "gmm":
+                        cluster_idx = len(dc.parent.children) - 1
+                        parent_model = dc.parent.model_
+                        model = GaussianMixture()
```

Comment:
@bdpedigo I made some simple changes below. Is it close to what we want? If not, I'll just take this part out for now as you suggested.

Reply:
this seems more in line with what i was thinking, I'm just going to merge for now and we can improve later if need be. have you tested this out at all?

Reply:
possibly something to mention/test in a tutorial but we can talk more later